data/sql/customizations.sql

```diff
@@ -35,4 +35,20 @@ INSERT INTO authority_to_authority_preference(source_auth_name, target_auth_name
     ('IGNF', 'EPSG', 'PROJ,IGNF,EPSG' );
 
 INSERT INTO authority_to_authority_preference(source_auth_name, target_auth_name, allowed_authorities) VALUES
-    ('ESRI', 'EPSG', 'PROJ,ESRI,EPSG' );
\ No newline at end of file
+    ('ESRI', 'EPSG', 'PROJ,ESRI,EPSG' );
+
+-- Custom ellipsoids (from proj -le)
+
+INSERT INTO "ellipsoid" VALUES('PROJ','ANDREA','Andrae 1876 (Den., Iclnd.)',NULL,'PROJ','EARTH',6377104.43,'EPSG','9001',300.0,NULL,0);
```

Comment:
ANDREA should be ANDRAE (or, in the ideal Unicode-everywhere world, "ANDRÆ")

Reply:
For the sake of simplicity, let's stick to ASCII character set for identifiers...

Reply:
While we're at it, why not expand the countries names fully? `Denmark, Iceland`. We can afford the few extra characters :-)

Reply:
Also, I think the modified Andræ (a.k.a. "Danish") is missing here:

    danish a=6377019.2563   rf=300.0         Andrae 1876 (Denmark, Iceland)

Included since PROJ 5.(0?)

Based on a revised conversion factor between fathoms and meters, I believe - I'll have to look it up, but cannot remember from where I got that material.

Reply:
> While we're at it, why not expand the countries names fully? `Denmark, Iceland`. We can afford the few extra characters :-)

Done -- to be honest, I didn't even realise that was what Den. represented !

Reply:
> Also, I think the modified Andræ (a.k.a. "Danish") is missing here:
> 
> ```
> danish a=6377019.2563   rf=300.0         Andrae 1876 (Denmark, Iceland)

It's already present in the database, from the EPSG db -- EPSG:7051, "Danish 1876"